docs/reference/directives.md

```
@@ -217,10 +217,19 @@ it("description with {0} and {1}", (param1: type, param2: type):
 )
 ```
 
+The argument can be any expression that evaluates to a list of tuples, including a function call:
+
+```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)